app/land_use.py

```diff
@@ -0,0 +1,69 @@
+import os
+import pandas as pd
+
+
+class LandUse:
+    technologies = []
+    commodities = []
+
+    def __init__(self, config):
+        self.config = config
+        self.__parse_file()
+
+    def regions(self):
```

Comment:
By making `LandUse` into a class, we can now divide the code in different small parts.